pvlib/irradiance.py

```diff
@@ -630,9 +643,20 @@ def isotropic(surface_tilt, dhi):
        Energy vol. 201. pp. 8-12
        :doi:`10.1016/j.solener.2020.02.067`
     '''
-    sky_diffuse = dhi * (1 + tools.cosd(surface_tilt)) * 0.5
+    poa_sky_diffuse = dhi * (1 + tools.cosd(surface_tilt)) * 0.5
 
-    return sky_diffuse
+    if return_components:
+        diffuse_components = OrderedDict()
```

Comment:
Minor note: no need to use OrderedDict in new code (https://github.com/pvlib/pvlib-python/issues/1684)